extensions/positron-python/src/client/positron/interpreterSettings.ts

```diff
@@ -209,3 +210,27 @@ export function printInterpreterDebugInfo(interpreters: PythonEnvironment[]): vo
     traceInfo('================ [END] PYTHON INTERPRETER DEBUG INFO ================');
     traceInfo('=====================================================================');
 }
+
+/**
+ * Retrieves the user's default Python interpreter path from VS Code settings
+ *
+ * @returns The configured Python interpreter path if it exists and is not 'python',
+ *          otherwise returns an empty string
+ */
+export function getUserDefaultInterpreter(scope?: Resource): InspectInterpreterSettingType {
+    const configuration = vscode.workspace.getConfiguration('python', scope);
```

Comment:
Suggestion to use `getConfiguration('python')` here to match the other uses in this file

```suggestion
    const configuration = getConfiguration('python', scope);
```